src/components/SelectionPane.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "styled-components";
+
+import CollapsableHeader from "./CollapsableHeader";
+import SelectionCard from "./SelectionCard";
+
+import { getFaction } from "util/jsonUtils";
+
+const Body = styled.div`
+  border-style: solid;
+  border-width: 1px;
+  flex-grow: 1;
+  border-right: 0;
+`;
+
+const Category = ({ title, units }) => (
+  <CollapsableHeader key={title} title={title}>
+    {units.map(subitem => {
+      return <SelectionCard unit={subitem} key={subitem.name} />;
+    })}
```

Comment:
Can be an implicit return:
```suggestion
    {units.map(subitem => <SelectionCard unit={subitem} key={subitem.name} /> )}
```

Reply:
And don't forget your key props! 🔑